src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__button {
+    display: block;
+    padding: 12px 0;
+    border-radius: 5px;
+    background-color: #00acdc;
+
+    color: white;
+    text-decoration: none;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
```

Comment:
I forgot to mention to add a small transition of the button color, try adding if you have extra time or when you assemble all the project, for now it's okay